webapp/components/features/FollowList/FollowList.vue

```diff
@@ -0,0 +1,157 @@
+          :loading="this.isLoading"
+          size="small"
+          color="softer"
+        >
+          {{
+            $t('profile.network.andMore', {
+              number: this.allConnectionsCount - this.connections.length,
+            })
+          }}
+        </base-button>
+      </ds-space>
+    </template>
+    <template v-else-if="this.connections.length > 7">
+      <div class="overflow-container">
+        <ds-space
+          v-for="follow in uniq(this.filteredConnections)"
+          :key="follow.id"
+          margin="x-small"
+        >
+          <client-only>
+            <user-teaser :user="follow" />
+          </client-only>
+        </ds-space>
+      </div>
+      <ds-space margin="x-small">
+        <ds-input
+          @input.native="setFilter"
+          :placeholder="filter"
+          v-focus="true"
+          size="small"
+          icon="filter"
+        />
+      </ds-space>
+    </template>
+  </base-card>
+</template>
+
+<script>
+import uniqBy from 'lodash/uniqBy'
+import UserTeaser from '~/components/UserTeaser/UserTeaser'
+import { followedByQuery, followingQuery } from '~/graphql/User'
+
+export default {
+  name: 'FollowerList',
+  components: {
+    UserTeaser,
+  },
+  props: {
+    user: { type: Object, default: null },
+    type: { type: String, default: 'following' },
+  },
+  data() {
+    return {
+      additionalConnections: [],
+      filter: null,
+      isLoading: false,
+      queries: {
+        followedBy: followedByQuery,
+        following: followingQuery,
+      },
+    }
+  },
+  computed: {
+    userName() {
+      const { name } = this.user || {}
+      return name || this.$t('profile.userAnonym')
+    },
+    connections() {
+      return [...this.user[this.type], ...this.additionalConnections]
+    },
+    filteredConnections() {
+      if (!this.filter) {
+        return this.connections
+      }
+
+      const fuzzyExpression = new RegExp(
+        `${this.filter.split('').reduce((part, c) => `${part}[^${c}]*${c}`)}`,
+        'i',
+      )
+
+      const fuzzyScores = this.connections
+        .map((user) => {
+          const match = user.name.match(fuzzyExpression)
+
+          return {
+            user,
+            score: match ? match[0].length * (match.index + 1) : -1,
+          }
+        })
+        .filter((score) => score.score !== -1)
+        .sort((a, b) => a.score - b.score)
+
+      return fuzzyScores.map((score) => score.user)
+    },
+    allConnectionsCount() {
+      return this.user[`${this.type}Count`]
+    },
+  },
+  methods: {
+    uniq(items, field = 'id') {
+      return uniqBy(items, field)
+    },
+    async fetchConnections() {
+      this.$set(this, 'isLoading', true)
+      const { data } = await this.$apollo.query({
+        query: this.queries[this.type],
+        variables: { id: this.user.id },
+        // neither result nor update are being called when defined here (?)
+      })
+      this.additionalConnections = data.User[0][this.type]
```

Comment:
I think we could use the same query for `User` and update the apollo cache by using `updateQuery`
there are some examples of this, that have some differences, but you could check them out, or we could look together. For example, `/pages/index.vue`
Also, see https://vue-apollo.netlify.com/api/smart-query.html#fetchmore
It would be nice to take advantage of some of the power of `smart` queries, reactive variables, and apollo caching to avoid adding extra queries.

Reply:
done in e6361c9